src/components/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -0,0 +1,25 @@
+import React from from 'react';
+import { render, screen } from '@testing-library/react';
+import Breadcrumbs from './Breadcrumbs';
+
+describe('#Breadcrumbs', () => {
+  it('should render breadcrumbs correctly', async () => {
+    render(
+      <Breadcrumbs data-testid="breadcrumbs">
+        <p> Item 1 </p>
```

Comment:
Probably should be using sub components for this.

like...

```js
<Breadcrumbs data-testid="breadcrumbs">
  <Breadcrumbs.Item>Item 1</Breadcrumbs.Item>
  <Breadcrumbs.Item>Item 2</Breadcrumbs.Item>
</Breadcrumbs>
```

Reply:
😅  don't know why I didn't think of doing it this way - it's so much cleaner. I'll make the changes, thanks @MildTomato

Reply:
There might also be some correct markup for breadcrumbs.
Only did a quick search but the example in here shows it in a `<ol>` tag, along with `aria-current`.

`current` (or `active` 🤷‍♂️ ) might be a good prop to have on <Breadcrumbs.Item> so you can have `aria-current` set properly